parm/soca/obsprep/obsprep_config.yaml

```diff
@@ -153,17 +153,17 @@ observations:
     name: insitu_profile_bathy
     provider: GTS
     dmpdir subdir: atmos
-    output file: '*.bathy_profile.ioda.nc'
-    dmpdir regex: 'bathy.*.dcom_subsampled'
+    output file: 'bathy_profile.ioda.nc'
```

Comment:
If these are supposed to be the template for the IODA file filenames, I don't think they'll work in the subsequent tasks - the netcdf to IODA example generates the IODA file name from the obs space name - I'll put a note where

Reply:
And yes, the ones I left in there are superfluous

Reply:
@AndrewEichmann-NOAA the `output file` name is not used from our conversation last time.
We can get rid of this yaml key...
My question: is this used for netcdf as well?

Reply:
I will get rid of the `output file` yaml key and as discussed will add a yaml key `type` and have an entry `nc or bufr/BUFR` to distinguish between the types.